### PR TITLE
Engine install freshness

### PR DESCRIPTION
**Pull request by stephaurelio**
### Proposed changes

Freshness updates for Docker Engine installation docs

### CODE REVIEW

File: content/includes/engine-install-methods-apt.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+You can install Docker Engine in different ways, depending on your needs:
+
+- Docker Engine comes bundled with
+  [Docker Desktop for Linux](../../desktop/install/linux-install.md). This is
+  the easiest and quickest way to get started.
+
+- Set up and install Docker Engine from
+  [Docker's Apt repository](#install-using-the-repository).
```
Comment (stephaurelio) [inline content/includes/engine-install-methods-apt.md:8]: Note for reviewer: We inconsistently say `apt` and Apt in our docs, and I've seen it multiple ways in other docs. Thoughts on how we want to standardize this here?

File: content/engine/install/centos.md [hunk 2]
```diff
@@ -33,12 +33,11 @@ CentOS versions:
 - CentOS 9 (stream)
 
 The `centos-extras` repository must be enabled. This repository is enabled by
-default, but if you have disabled it, you need to
-[re-enable it](https://wiki.centos.org/AdditionalResources/Repositories).
```
Comment (stephaurelio) [inline content/engine/install/centos.md:37]: Note for reviewer: this link is a 404 and didn't include much helpful info about this topic. I opted to not explain how to do this, but wanted feedback on if we should include the commands for how to enable repositories using `yum` or `dnf`.

Comment (stephaurelio) [reply content/engine/install/centos.md:37]: Related issue: https://github.com/docker/docs/issues/18364

Comment (dvdksn) [reply content/engine/install/centos.md:37]: I'm OK with removing the link!

File: content/includes/engine-install-methods-apt.md (hunk 1 above)
Comment (dvdksn) [reply content/includes/engine-install-methods-apt.md:8]: I've noticed this as well and deferred making my mind up 🙈 This is an acronym for Advanced Packaging Tool, so technically we should use APT. But you only really care about this tool as a CLI invocation, much like `curl`. So IMO, we're better off using `apt` for this (and similar utilities) - let's do that

Comment (dvdksn) [reply content/includes/engine-install-methods-apt.md:8]: Apt is definitely wrong and my bad - not sure what I was thinking

**Review by dvdksn — CHANGES_REQUESTED** [review]
I'd prefer if we don't move the install methods to an include. While reuse is good, these docs get a lot of contributions, and the include complicates this.

File: content/includes/engine-install-methods-apt.md (hunk 1 above)
Comment (stephaurelio) [reply content/includes/engine-install-methods-apt.md:8]: K will update to use `apt` 👍

File: content/desktop/install/debian.md [hunk 3]
```diff
@@ -50,8 +50,8 @@ To install Docker Desktop successfully, you must:
 
 Recommended approach to install Docker Desktop on Debian:
 
-1. Set up Docker's Apt repository.
-   See [Install using the Apt repository, step 1](../../engine/install/debian.md#install-using-the-repository).
+1. Set up Docker's 'apt' repository.
+   See [Install using the 'apt' repository, step 1](../../engine/install/debian.md#install-using-the-repository).
```
Comment (dvdksn) [inline content/desktop/install/debian.md:54]:
```suggestion
1. Set up Docker's `apt` repository.
   See [Install using the `apt` repository, step 1](../../engine/install/debian.md#install-using-the-repository).
```

**Review by dvdksn — APPROVED** [review]
LGTM thanks!

**Pull request merged**